x/incentives/keeper/genesis_test.go

```diff
@@ -27,43 +95,75 @@ func TestIncentivesExportGenesis(t *testing.T) {
 
 	// create an address and fund with coins
 	addr := sdk.AccAddress([]byte("addr1---------------"))
-	coins := sdk.Coins{sdk.NewInt64Coin("stake", 10000)}
+	coins := sdk.Coins{sdk.NewInt64Coin("stake", 20000), sdk.NewInt64Coin("uosmo", 10000000000)}
 	err := simapp.FundAccount(app.BankKeeper, ctx, addr, coins)
 	require.NoError(t, err)
 
+	// allow pool creation
+	clParams := app.ConcentratedLiquidityKeeper.GetParams(ctx)
+	clParams.IsPermissionlessPoolCreationEnabled = true
+	app.ConcentratedLiquidityKeeper.SetParams(ctx, clParams)
+
+	// create two pools to be used for group creation
+	msgCreatePool := model.MsgCreateConcentratedPool{
+		Sender:       addr.String(),
+		Denom0:       "uion",
+		Denom1:       "uosmo",
+		TickSpacing:  100,
+		SpreadFactor: osmomath.MustNewDecFromStr("0.0005"),
+	}
+	_, err = app.PoolManagerKeeper.CreatePool(ctx, msgCreatePool)
+	require.NoError(t, err)
+	_, err = app.PoolManagerKeeper.CreatePool(ctx, msgCreatePool)
+	require.NoError(t, err)
```

Comment:
nit: there is a helper `s.PrepareAllSupportedPools()` that might be helpful here

Reply:
we are using t testing over testsuite and would rather not refactor it to allow for this.